nltk/classify/api.py

```diff
@@ -38,12 +43,13 @@ class ClassifierI(object):
     Subclasses may define:
       - either ``prob_classify()`` or ``prob_classify_many()`` (or both)
     """
+    @abstractmethod
     def labels(self):
         """
         :return: the list of category labels used by this classifier.
         :rtype: list of (immutable)
         """
-        raise NotImplementedError()
+        pass
```

Comment:
Oh yeah, thanks. Actually I was just figuring out how to reconcile the "hard" enforcement of `abc.abstractmethod` with how how the `ClassifierI` ABC is implemented - see 121a24ca66d2913d1567e7c9e7e26af6a78c3afc